services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/discussion/DiscussionThreadManagerImplTest.java

```diff
@@ -105,7 +108,7 @@
 	private List<DiscussionThreadEntityReference> entityRefs = new ArrayList<DiscussionThreadEntityReference>();
 	private List<DiscussionThreadEntityReference> titleEntityRefs = new ArrayList<DiscussionThreadEntityReference>();
 
-	@Before
+	@BeforeEach
 	public void before() {
 		MockitoAnnotations.initMocks(this);
```

Comment:
We don't need that, we can just use the @InjectMocks annotations

Reply:
Done.